deeplake/core/vector_index/factory.py

```diff
@@ -0,0 +1,37 @@
+from .indexer import Indexer
+from .mutable_indexer import MutableIndexer
+
+from typing import Callable, Dict, Tuple, Any, Optional, Iterable
+
+
+class IndexFactory:
```

Comment:
Where `IndexFactory` is used? Do we need it?

Reply:
These remained here from the initial prototyping. Removed all this stuff.